src/main/kotlin/racing/Car.kt

```diff
@@ -1,8 +1,18 @@
 package racing
 
-class Car(var position: String = "") {
+class Car(val name: String, var position: Int) {
+
+    constructor(name: String) : this(name, 0)
```

Comment:
부생성자를 사용하셨군요?  단순히 프로퍼티에 default 값을 주입하기 위함이라면 kotlin에서는 [default Argument](https://kotlinlang.org/docs/functions.html#default-arguments)를 활용하면 별도의 부생성자를 만들 필요가 없어요. 😉

```suggestion
class Car(val name: String, var position: Int = 0) {
```